test/lightning/attempts/prom_ex_plugin_test.exs

```diff
@@ -33,4 +36,114 @@ defmodule Lightning.Attempts.PromExPluginText do
     assert metric.tags == []
     assert metric.unit == :millisecond
   end
+
+  test "polling_metrics returns a Promex Polling instance" do
+    stalled_attempt_threshold_seconds =
+      Application.get_env(:lightning, :metrics)[
+        :stalled_attempt_threshold_seconds
+      ]
+
+    expected_mfa =
+      {
+        Lightning.Attempts.PromExPlugin,
+        :stalled_attempt_count,
+        [stalled_attempt_threshold_seconds]
+      }
+
+    polling = PromExPlugin.polling_metrics(%{})
+
+    assert polling.group_name == :lightning_attempt_polling_events
+
+    assert polling.poll_rate == 5000
+
+    assert polling.measurements_mfa == expected_mfa
+
+    [metric | _] = polling.metrics
+
+    assert %Telemetry.Metrics.LastValue{} = metric
```

Comment:
Would do the same here even though it is a small code. It's one less check for the brain to know if the assertions have something in common or not.